data/governments.txt

```diff
@@ -98,6 +98,20 @@ government "Hai"
 	"hostile hail" "hostile hai"
 	"hostile disabled hail" "hostile disabled hai"
 
+government " Hai "
+	swizzle 0
+	color .84 .42 .81
+	
+	"player reputation" 0
+	"attitude toward"
```

Comment:
This needs attitude toward `"Hai" 1` and `"Hai"` needs `" Hai " 1` so that their reputations will match and attacking one angers the other.